specification/authorization/resource-manager/Microsoft.Authorization/preview/2018-05-01-preview/authorization-AccessReviewCalls.json

```diff
@@ -0,0 +1,1981 @@
+            "modelAsString": true
+          }
+        },
+        "startDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The DateTime when the review instance is scheduled to be start.",
+          "x-nullable": false
+        },
+        "endDateTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The DateTime when the review instance is scheduled to end.",
+          "x-nullable": false
+        }
+      },
+      "description": "Access Review Instance properties."
+    },
+    "AccessReviewInstanceStatistics": {
+      "properties": {
+        "totalCount": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The total count of instance statistics."
+        },
+        "decisionsSummary": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AccessReviewInstanceDecisionsSummary",
+          "description": "Decisions summary."
+        },
+        "recommendationsSummary": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AccessReviewInstanceRecommendationsSummary",
+          "description": "Decisions summary."
+        }
+      },
+      "description": "Access Review Instance properties."
+    },
+    "AccessReviewInstanceDecisionsSummary": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccessReviewInstanceDecisionCount"
+          },
+          "description": "Access Review Instance Decision Summary list."
+        }
+      },
+      "description": "List of Access Review decision counts."
+    },
+    "AccessReviewInstanceRecommendationsSummary": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccessReviewInstanceRecommendationCount"
+          },
+          "description": "Access Review Instance Recommendations Summary list."
+        }
+      },
+      "description": "List of Access Review recommendations count."
+    },
+    "AccessReviewInstanceDecisionCount": {
+      "properties": {
+        "count": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The count of records with decision = {decision}."
+        },
+        "decision": {
+          "type": "string",
+          "description": "The reviewer's decision type. This can be one of NotReviewed, Approve and Deny",
+          "enum": [
+            "Approve",
+            "Deny",
+            "NotReviewed",
+            "DontKnow",
+            "NotNotified"
+          ],
+          "x-ms-enum": {
+            "name": "AccessReviewInstanceDecisionType",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Access Review Instance Decision Count."
+    },
+    "AccessReviewInstanceRecommendationCount": {
+      "properties": {
+        "count": {
```

Comment:
>count [](start = 9, length = 5)

Is this readonly?

Reply:
Yes, all of these properties are read only